### DISCUSSION TITLE
Integration of TMS Sphinx

### DISCUSSION THREAD

**Discussion by cbarthol** [holgerflick/flixaccounting#34] [Ideas]
HI,
i would love to see how to integrate TMS Sphinx for the user accounts

**Comment by holgerflick** [comment]
Excellent suggestion! Thank you. TMS Sphinx is actually on "my radar" for future tutorials. I am just waiting for the next release as it has been a while and I don't want to start creating something when a new release is most likely around the corner. I will talk to TMS. 

@wlandgraf
